### PR TITLE
fixes #5832 - org switcher does not collapse when moving mouse diagnally to submenu

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by isratrade** [comment]
@abenari, can you review? any ideas for re-factoring?

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -21,6 +21,17 @@ $(function() {
...
+  // keep submenu for loc/org picker open when mouse is moved diagnally and not hoving over menu
+  $('.org-submenu').on('mouseover', function(){
+    $('.org-menu').show();
+    $('.loc-menu').hide();
+  });
+  $('.loc-submenu').on('mouseover', function(){
+    $('.loc-menu').show();
+    $('.org-menu').hide();
+  });
+
```
Comment (abenari) [inline app/assets/javascripts/application.js]: 1. I think the right place for this code is in the topbar.js
2. I would have used document level events and not on-load event registration e.g.

``` javascript
// keep submenu for loc/org picker open when mouse is moved diagonally and not hovering over menu
$(document).on('mouseover','.org-submenu', function(){
    $('.org-menu').show();
    $('.loc-menu').hide();
});
$(document).on('mouseover','.loc-submenu', function(){
    $('.loc-menu').show();
    $('.org-menu').hide();
});
```

Comment (abenari) [inline app/assets/javascripts/application.js]: also adding 

``` javascript
$(document).on('mouseleave','.org-menu', function(){
    $('.org-menu').hide();
});
$(document).on('mouseleave','.loc-menu', function(){
    $('.loc-menu').hide();
});
```

makes the org-switcher more consistent with the rest of the menu

Comment (abenari) [inline app/assets/javascripts/application.js]: and last comment, I think the names are misleading org-submenu should be org-menu and visa-versa.

**Comment by isratrade** [comment]
@abenari, updated per your suggested changes. thanks!

**Comment by GregSutcliffe** [comment]
Tested, seems to solve the problem perfectly. :+1: to merge

**Comment by domcleal** [comment]
Merged as 6addb89 for Foreman 1.6.0, thanks @isratrade.

**Pull request closed without merging**